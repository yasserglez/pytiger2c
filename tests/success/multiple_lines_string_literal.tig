let
    var s := "Tiger allows string constants to \
             \span multiple lines by ending and \
             \starting each line with a backslash."
in
    print(s)
    print("\n")
end