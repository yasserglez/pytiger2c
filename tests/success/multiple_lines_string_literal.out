Tiger allows string constants to span multiple lines by ending and starting each line with a backslash.
